### ISSUE TITLE
Android API version 26 and 27

### ISSUE THREAD

**Issue by krokhale** [callstack/react-native-opentok#71]
I can confirm that this works on API version 23,24 and 25. And it's crashing on 26 and 27.

**Comment by krokhale** [comment]
According to your readme:

Supported OpenTok SDK version:

OpenTok SDK 2.11.+

I believe opentok just added support for Oreo(26 and 27) in 2.12+, is there an upgrade path for react-native-opentok to start using 2.12+

<img width="458" alt="screen shot 2017-11-24 at 4 01 46 pm" src="https://user-images.githubusercontent.com/173148/33206650-ef460890-d130-11e7-900c-a1fb0422bb03.png">

**Comment by krokhale** [comment]
Perfect thanks! Though i did manage to find a workaround for now. Will update whenever you folks merge.

**Comment by dratwas** [comment]
@krokhale It's merged. could you check if it works for you?

**Comment by krokhale** [comment]
Sorry just getting to this now, tested on API 26 and 27, the video call starts and it does not crash on dev. Though it crashes on call disconnect.  Could be related to this issue, since we have only tested  with Oreo 8 on dev:

https://github.com/callstack/react-native-opentok/issues/76

We have just pushed out a new version to production. Will post a comment here if it crashes after disconnect on production too.


Also to anyone reading this, in order to update your react-native-opentok use this:

yarn add https://github.com/callstack/react-native-opentok

instead of

yarn add react-native-opentok

Thanks for your work!
